### PR TITLE
Azure Monitor Exporter - Add MetricHelperTests

### PR DESCRIPTION
**Pull request by rajkumar-rangaraj**

### CODE REVIEW

**Review by vishweshbankwar — APPROVED** [review]

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/MetricHelperTests.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    public class MetricHelperTests
+    {
+        [Fact]
+        public void ValidateMetricTelemetryItem()
+        {
+            var metrics = new List<Metric>();
+
+            using var meter = new System.Diagnostics.Metrics.Meter(nameof(ValidateMetricTelemetryItem));
+            using var provider = Sdk.CreateMeterProviderBuilder()
+                .AddMeter(meter.Name)
```
Comment (cijothomas) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/MetricHelperTests.cs:22]: where is this `meter` defined?

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/MetricHelperTests.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            Assert.Equal("00000000-0000-0000-0000-000000000000", telemetryItems[0].InstrumentationKey);
+            Assert.Equal("testRoleName", telemetryItems[0].Tags[ContextTagKeys.AiCloudRole.ToString()]);
+            Assert.Equal("testRoleInstance", telemetryItems[0].Tags[ContextTagKeys.AiCloudRoleInstance.ToString()]);
+
+            var metricsData = (MetricsData)telemetryItems[0].Data.BaseData;
+            Assert.Equal(2, metricsData.Version);
+            Assert.Equal("TestDoubleCounter", metricsData.Metrics.First().Name);
+            Assert.Equal(123.45, metricsData.Metrics.First().Value);
+            Assert.Equal(DataPointType.Aggregation, metricsData.Metrics.First().DataPointType);
+            Assert.Equal(1, metricsData.Properties.Count);
```
Comment (cijothomas) [inline sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/MetricHelperTests.cs:42]: validate the actual property as well.

**Review by cijothomas — APPROVED** [review]

File: sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/MetricHelperTests.cs (hunk 1 above)
Comment (rajkumar-rangaraj) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/MetricHelperTests.cs:23]: It is defined in line 21.

Comment (cijothomas) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/MetricHelperTests.cs:23]: 😄 sorry i missed !

Comment (cijothomas) [reply sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Tests/MetricHelperTests.cs:23]: Good reminder to schedule annual eye check appointment :)

**Pull request merged**